target/linux/ath79/image/tiny.mk

```diff
@@ -30,3 +30,13 @@ define Device/buffalo_whr-g301n
   SUPPORTED_DEVICES += whr-g301n
 endef
 TARGET_DEVICES += buffalo_whr-g301n
+
+define Device/pqi_air-pen
+  ATH_SOC := ar9330
+  DEVICE_TITLE := PQI Air-Pen
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage
```

Comment:
we usually don't include the `kmod-usb-storage` in the image. It is up to the user to download and install this package. Furthermore, without a filesystem kmod, it doesn't make much sense to include the storage module.